byway/pom.xml

```diff
@@ -43,17 +43,6 @@
       <version>1.9.59</version>
     </dependency>
 
-    <dependency>
-      <groupId>com.google.code.gson</groupId>
-      <artifactId>gson</artifactId>
-      <version>2.8.6</version>
-    </dependency>
-
-    <dependency>
-      <groupId>com.google.appengine</groupId>
-      <artifactId>appengine-api-1.0-sdk</artifactId>
-      <version>1.9.59</version>
-    </dependency>
```

Comment:
You don't have to delete dependencies you don't use in this branch, since they are still in the master branch and used for other files.

Reply:
They are still there. I just deleted duplicates